spec/Section 5 -- Validation.md

```diff
@@ -1,8 +1,8 @@
 # Validation
 
-GraphQL does not just verify if a request is syntactically correct, but also
-ensures that it is unambiguous and mistake-free in the context of a given
-GraphQL schema.
+A GraphQL service does not just verify if a request is syntactically correct,
```

Comment:
Perhaps: A GraphQL “Validator” instead of the more generic “service”?

Reply:
I mean just that not all services perform validation... I am not sure about the exact language (or capitalization)... maybe just "A conforming validator..."

Reply:
Meaning a service can be structured so that all validation is performed at build time. Anyway agree this should be separate PR

Reply:
similar to other case, 'service' is used throughout; if we want to introduce validator or executor, this should be a separate PR changing it here and in other places; and by the way explaining what things are (validator, executor).